config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  root 'application#hello'
-end
\ No newline at end of file
+  scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
+    root "static_pages#home"
+    get "static_pages/home"
+    get "static_pages/help"
+    get  "static_pages/about"
+    get  "static_pages/contact"
```

Comment:
convention
mỗi từ cách nhau 1 khoanrg trắng thôi
```suggestion
    get "static_pages/contact"
```